### PR TITLE
+mparser.1.2.1

### PR DESCRIPTION
**Pull request by cakeplus**
Release notes: https://github.com/cakeplus/mparser/releases/tag/1.2.1.

### CODE REVIEW

**Comment by cakeplus** [comment]
Why did @camelus mark a package as **not installable**, while the CI managed to install it just fine?

**Comment by samoht** [comment]
Not sure what happened on @camelus but merging as everything went well on the CI. @AltGr any idea?

File: packages/mparser/mparser.1.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "Max Mouratov <mmouratov@gmail.com>"
+authors: "Holger Arnold <holger@harnold.org>"
+homepage: "https://github.com/cakeplus/mparser/"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "git://https://github.com/cakeplus/mparser"
+bug-reports: "https://github.com/cakeplus/mparser/issues"
+depends: [
+  "ocamlfind"
```
Comment (samoht) [inline packages/mparser/mparser.1.2.1/opam]: Should `ocamlfind` have also a`{build}` flag? If yes, could you add it? Thanks!

Comment (cakeplus) [inline packages/mparser/mparser.1.2.1/opam]: Done!

**Comment by cakeplus** [comment]
Strangely, @camelus complains again.

**Comment by cakeplus** [comment]
Bump. Can this be merged now? The new hashes are needed in OPAM.

**Comment by samoht** [comment]
Woops, sorry for the delay. Merged!

**Comment by cakeplus** [comment]
Thank you! \o/

**Pull request merged**